harbor-client/src/lib.rs

```diff
@@ -436,13 +436,25 @@ impl HarborCore {
         invite_code: InviteCode,
     ) -> anyhow::Result<ClientConfig> {
         let download = Instant::now();
-        let config = fedimint_api_client::api::net::Connector::Tor
-            .download_from_invite_code(&invite_code)
-            .await
-            .map_err(|e| {
-                error!("Could not download federation info: {e}");
-                e
-            })?;
+        let config = {
+            #[cfg(feature = "disable-tor")]
+            let config = fedimint_api_client::api::net::Connector::Tcp
+                .download_from_invite_code(&invite_code)
+                .await
+                .map_err(|e| {
+                    error!("Could not download federation info: {e}");
+                    e
+                })?;
+            #[cfg(not(feature = "disable-tor"))]
```

Comment:
can we have these include the `debug_assertions` like I did in the other one, that way for a production release we can't accidentally disable tor